sdk/readbackspan.go

```diff
@@ -0,0 +1,19 @@
+package sdk
+
+import "context"
+
+// ReadbackSpan is a span whose attributes can be read. It is important
+// to bear in mind that only those attributes being recorded through this
+// interface will be available for read. This is, an existing span where
+// attributes have been already recorded can be turned into a ReadbackSpan
+// but the attributes won't be available.
+type ReadbackSpan interface {
+	Span
+	GetAttribute(key string) interface{}
```

Comment:
Also need a method to get all attributes.

Reply:
I am curious about the reason, this will be called inside the filter and I assume filter knows what attributes to read, isn't?

Reply:
We moved that further into libtraceable so that we didn't have to keep updating all the libtraceable users (ext_cap, nginx, envoy, etc) if the required attributes change. At the moment, we would send all span attributes.

Reply:
Then I would change this signature into return all attributes.